Sources/BBCodeParser.php

```diff
@@ -2730,7 +2730,7 @@ protected function parseMessage(): void
 				$this->last_pos = max($this->last_pos, 0);
 
 				// Pick a block of data to do some raw fixing on.
-				$data = substr($this->message, $this->last_pos, $this->pos - $this->last_pos);
+				$data = substr($this->message, $this->last_pos, (int) ($this->pos - $this->last_pos));
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.